src/test/java/pages/HomePage.java

```diff
@@ -1,20 +1,59 @@
 package pages;
-
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import java.util.List;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.PageFactory;
 
 public class HomePage extends BasePage {
-
-    By userAvatarIcon = By.cssSelector("img.avatar");
-
-    public HomePage( WebDriver givenDriver) {
-
-        super(givenDriver);
+    String playlistName = "my";
+    String newPlaylistName = "lalala";
+    @FindBy(xpath = "//*[@title='Create a new playlist']")
+    WebElement addNewPlaylist;
+    @FindBy(xpath = "//*[text()='New Playlist']")
+    WebElement newPlaylist;
+    @FindBy(xpath = "//input[@name='name']")
+    WebElement nameField;
+    @FindBy(xpath = "//li/a[text()='my']")
+    WebElement myPlaylist;
+    @FindBy(xpath = "//li[text()='Edit']")
+    WebElement editButton;
+    @FindBy(xpath = "//input[@name='name']")
+    WebElement editField;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");
    
```
   @FindBy(xpath = "//*[@title='Create a new playlist']")
    private WebElement addNewPlaylist;
```